catalog/ui/search-ui/simple/src/main/resources/SearchPage.jsp

```diff
@@ -62,23 +62,23 @@
 <body>
 
 	<div class="navbar navbar-inverse navbar-fixed-top">
-		<% String h = ConfigurationStore.getInstance().getHeader();
+		<% String h = props.getHeader();
 			if(h != null && h.trim().length() > 0)
```

Comment:
✏️ Don't think any of the `UiConfigutationProperties.get...()` can return `null`, so we probably don't need those `null` checks.

Reply:
Yes, I was actually just about to remove these!

Reply:
✏️  Just a note here: I'm going to be getting rid of this whole println thing as well. 
This gets weird as far as sanitizing the inputs goes and I don't think this is the best way to conditionally add HTML to the page.